saleor/static/scss/components/_cards.scss

```diff
@@ -16,12 +13,14 @@
     }
     p {
       margin-bottom: 0;
+      padding: 1rem;
```

Comment:
I don't think that `<p>` should have padding anywhere. Do we actually use this tag here anyway?

Reply:
instead of directly padding `<p>` modify the wrapper class:
```
    .address-details{
      padding: $global-padding;
    }
```
Unfortunately i'm unable to have the changes reflected here, perhaps you could do it?